extensions/Skyhigh173/bigint.js

```diff
@@ -0,0 +1,456 @@
+(function(Scratch){
+  'use strict';
+
+  const bi = x => {
+    try {
+      if (x.slice(-1) == 'n') {
+        return BigInt(x.slice(0, -1));
+      }
+      return BigInt(x);
+    } catch {
+      try {
+        return BigInt(Math.trunc(Number(x)));
+      } catch {
+        return 0n;
+      }
+    }
+  };
+
+  const makeLabel = (text) => ({
+    blockType: 'label',
+    text: text
+  });
+
+
+  class BigIntExtension {
+    getInfo() {
+      return {
+        id: 'skyhigh173BigInt',
+        name: 'BigInt',
+        color1: '#59C093',
+        blocks: [
+          {
+            opcode: 'bigint_from',
```

Comment:
you can leave it as-is, but you don't *need* to put bigint_ in the opcodes; they are already prefixed with the extension ID so names like "from" are already guaranteed to not conflict with other extensions